src/lib/Handset/CRSFHandset.cpp

```diff
@@ -268,14 +265,26 @@ void CRSFHandset::RcPacketToChannelsData() // data is packed as 11 bits per chan
         bitsMerged -= srcBits;
     }
 
-    if (prev_AUX1 != ChannelData[4])
-    {
-        #if defined(PLATFORM_ESP32)
-        devicesTriggerEvent();
-        #endif
-    }
+    //
+    // sends channel data and also communicates commanded armed status in arming mode Function.
+    // frame len 24 -> arming mode Channel: use channel 5
+    // frame len 25 -> arming mode Function: use commanded arming status in extra byte
+    //
+    armCmd = inBuffer.asUint8_t[1] == 24 ? CRSF_to_BIT(ChannelData[4]) : payload[readByteIndex];
+
+    #if defined(PLATFORM_ESP32)
+        // monitoring arming state
+
+        static bool lastArmCmd = false;
```

Comment:
This is a class, store this in the class instance (make it a class private variable)

Reply:
done